packages/contact-center/cc-components/src/wc.ts

```diff
@@ -1,8 +1,14 @@
 import r2wc from '@r2wc/react-to-web-component';
 import UserStateComponent from './components/UserState/user-state';
+import OutdialCallComponent from './components/OutdialCall/out-dial-call';
 
 const WebUserState = r2wc(UserStateComponent);
+const WebOutDialCallComponent = r2wc(OutdialCallComponent);
```

Comment:
```suggestion
const WebOutdialCallComponent = r2wc(OutdialCallComponent);
```

Reply:
In fact, we can remove the word Component and have it as just this,
```suggestion
const WebOutdialCall = r2wc(OutdialCallComponent);
```

Reply:
Updated!